src/Model.php

```diff
@@ -43,6 +43,16 @@ abstract public function getKeyName();
      */
     abstract public function getColumns();
 
+    /**
+     * Get the configured table alias. (Default tablename)
```

Comment:
Please reference the method `getTableName` (with `{@see self::getTableName}`) instead of just `tablename`